scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/SynchronousSchedulerAgent.java

```diff
@@ -212,10 +217,26 @@ private Solver createScheduler(final PlanMetadata planMetadata, final Problem pr
    * @throws ResultsProtocolFailure when the requested plan cannot be loaded, or the target plan revision has
    *     changed, or aerie could not be reached
    */
-  private Plan loadInitialPlan(final PlanMetadata planMetadata, final Problem problem) {
+  private Triple<Plan, MerlinPlan, Map<SchedulingActivityInstanceId, ActivityInstanceId>> loadInitialPlan(final PlanMetadata planMetadata, final Problem problem) {
```

Comment:
I know we only use this `Triple` in one place, but I think it would help readability (on the caller's side) to define a record:

```
record PlanComponents(Plan schedulerPlan, MerlinPlan merlinPlan, Map<SchedulingActivityInstanceId, ActivityInstanceId> idMap) {}
```
It can go right below this function, next to the `SchedulerMissionModel` record, which serves a similar purpose.

This way, the caller code can refer to `loadedPlanComponents.idMap()` and  `loadedPlanComponents.merlinPlan()` instead of `loadedPlanComponents.getRight()` and  `loadedPlanComponents.getMiddle()`